docs/current/sdk/python/snippets/get-started/step1/test.py

```diff
@@ -22,7 +22,7 @@ async def test():
         )
 
         # execute
-        version = await python.stdout().contents()
+        version = await python.stdout()
```

Comment:
@aluzzardi Shouldn't we wait for a release before changing the docs?

Reply:
No we're good -- docs are not deployed on merge

/cc @vikram-dagger